pkg/scheduler/plugins/predicates/predicates.go

```diff
@@ -54,6 +63,53 @@ func formatReason(reasons []algorithm.PredicateFailureReason) string {
 	return strings.Join(reasonStrings, ", ")
 }
 
+type predicateEnable struct {
+	memoryPressureEnable bool
+	diskPressureEnable   bool
+	pidPressureEnable    bool
+}
+
+func enablePredicate(args framework.Arguments) predicateEnable {
+
+	/*
+		   User Should give predicatesEnable in this format(predicate.MemoryPressureEnable, predicate.DiskPressureEnable, predicate.PIDPressureEnable.
+		   Currently supported only for MemoryPressure, DiskPressure, PIDPressure predicate checks.
+
+		   actions: "reclaim, allocate, backfill, preempt"
+		   tiers:
+		   - plugins:
+		     - name: priority
+		     - name: gang
+		     - name: conformance
+		   - plugins:
+		     - name: drf
+		     - name: predicates
+		       arguments:
+		 		 predicate.MemoryPressureEnable: true
+		 		 predicate.DiskPressureEnable: true
+				 predicate.PIDPressureEnable: true
+		     - name: proportion
+		     - name: nodeorder
+	*/
+
+	predicate := predicateEnable{
+		memoryPressureEnable: false,
```

Comment:
Their default values are false.

Reply:
Yes, But Initializing those values is better, that is why have initialized it.

Reply:
I think `var predicate predicateEnable` is enough :) It is OK if you think we'd better initialize them explicitly.